modules/tiles/src/tileset/format-i3s/i3s-tileset-traverser.ts

```diff
@@ -1,34 +1,34 @@
 import {load} from '@loaders.gl/core';
-import TilesetTraverser from './tileset-traverser';
+import {TilesetTraverser} from '../tileset-traverser';
 
 import {getLodStatus} from '../helpers/i3s-lod';
-import TileHeader from '../tile-3d';
-import I3STileManager from './i3s-tile-manager';
+import {Tile3D} from '../tile-3d';
+import {I3STileManager} from './i3s-tile-manager';
 import {FrameState} from '../helpers/frame-state';
 
-export default class I3STilesetTraverser extends TilesetTraverser {
+export class I3STilesetTraverser extends TilesetTraverser {
   private _tileManager: I3STileManager;
 
+  constructor(options) {
+    super(options);
+    this._tileManager = new I3STileManager();
+  }
+
   /**
    * Check if there are no penging tile header requests,
    * that means the traversal is finished and we can call
    * following-up callbacks.
    */
-  protected traversalFinished(frameState: FrameState): boolean {
+  override traversalFinished(frameState: FrameState): boolean {
```

Comment:
The website cannot build with this modificator.
![image](https://user-images.githubusercontent.com/18549629/218498825-8adea8b5-6be7-4bc4-9f7d-30ae974b9978.png)